portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.data;
+
+public class Comment{
+    
```

Comment:
nit: get rid of extra blank line between the class declaration and the instance variables